communications/mqtt/publish/publish.ino

```diff
@@ -21,25 +21,31 @@ PubSubClient client(espClient);
 String jsonTopicStr;
 String newTopicStr;
 
+//Initialize time
```

Comment:
better writing small inline comments if it is not a section in your code

Reply:
OK!